parent/pom.xml

```diff
@@ -127,6 +127,8 @@
     <version.jackson-annotations>2.15.2</version.jackson-annotations>
     <version.swagger-annotations>2.2.15</version.swagger-annotations>
     <version.checker-qual>3.37.0</version.checker-qual>
+    <!-- java-jwt version is aligned with the one in identity-sdk -->
```

Comment:
🔧 This comment looks like it will quickly go outdated. It would also be interesting to understand why it is (or should be) aligned with identity-sdk.

❓ How would we verify this? Can we add a CI workflow runs once per day and fails when identity moves away from this version?

Reply:
I think it would be better to remove this comment.

There's no hard requirement to keep aligned with the `java-jwt` version used by the identity-sdk. We implemented the auth data interchange API to be pretty flexible on what library/mechanism we use for encoding. In the future, we may migrate to something completely unrelated to JWT. So this comment might even make things more confusing.